tool/tsh/proxy_test.go

```diff
@@ -76,6 +72,8 @@ func TestTSHSSH(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
+			t.Setenv("TELEPORT_HOME", t.TempDir())
```

Comment:
Can we use `HomeEnvVar` constant for TELEPORT_HOME?

Reply:
fixed!